### PR TITLE
4.2 Release Notes modified for dedicated

### PR DESCRIPTION
**Pull request by sheriff-rh**
Changes based on: https://docs.google.com/document/d/1th2emY24A7kgvTxUM5PUEd5-x0gzUH-EPEGyjjI1EFM/edit#

@wgordon17 PTAL
@patrickstrick  @vikram-redhat FYI
@ahardin-rh  would you mind peer reviewing? 
Local build: http://file.rdu.redhat.com/~antaylor/110419/4.2dedicatedRNs/release_notes/ocp-4-2-release-notes.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
If I recall correctly, we previously omitted the bug list and Tech Preview section because they are OCP-centric. Right now, the only new features and enhancements we highlight here are for Dev experience and Web console. I would expect there to be more, but I defer to Will.

**Comment by wgordon17** [comment]
@ahardin-rh Yeah, good call wrt TP section. @sheriff-rh can you please remove that section? Otherwise LGTM

I think the bug list is still beneficial for OSD customers to see what was fixed by the upgrade...in case they were waiting on something.

**Comment by wgordon17** [comment]
Also, any change we can get the left nav bar to agree with the topic header?

**Comment by sheriff-rh** [comment]
@wgordon17  I removed the TP table and that is reflected in the latest version of the build. I tried to change the title for the release notes page to `{product-title}`, which is populated per build version, but that broke things. I'll get with @ahardin-rh  and figure out a reasonable solution and title.

**Comment by wgordon17** [comment]
Awesome 👍🏻 Thanks!

File: _topic_map.yml [hunk 1]
```diff
@@ -39,12 +39,13 @@ Topics:
 ---
 Name: Release notes
 Dir: release_notes
-Distros: openshift-enterprise
+Distros: openshift-enterprise, openshift-dedicated
```
Comment (adellape) [inline _topic_map.yml]: @sheriff-rh The space here might be breaking it.

Comment (sheriff-rh) [reply _topic_map.yml]: good catch @adellape  ! just updated.

**Comment by sheriff-rh** [comment]
Closed in favor of #17888 .

**Pull request closed without merging**